docs/requirements.txt

```diff
@@ -0,0 +1,6 @@
+Sphinx
```

Comment:
You might want to `pip freeze` to get exact versions here. We've had problems in the past with compatibility for some of these libraries.